sonar-python-plugin/src/test/resources/org/sonar/plugins/python/ipynb/file1.ipynb

```diff
@@ -0,0 +1,24 @@
+# Integers
```

Comment:
Do we need all this code snippet? can we maybe simplify it a bit?

Reply:
It's a copy of the one used for the `PythonSensorTest`. I want to ensure the `IPynbSensor` works correctly on regular Python files. I think it would make sense to update this snippet (or even have a separate file) to have some IPython specific syntax once we start to support it.